oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/indexstore/IndexStoreUtils.java

```diff
@@ -39,6 +40,23 @@
 public class IndexStoreUtils {
     public static final String METADATA_SUFFIX = ".metadata";
 
+    public static final String OAK_INDEXER_USE_ZIP = "oak.indexer.useZip";
+    public static final String OAK_INDEXER_USE_LZ4 = "oak.indexer.useLZ4";
+
+    public static boolean compressionEnabled() {
```

Comment:
I think this is slightly ambiguous. It seems like USE_ZIP has two meanings: whether to enable compression **and** use GZIP to compress? But LZ4 only indicates whether we use LZ4. So if we disable ZIP but enable LZ4, it wouldn't be used. On the other hand, if we enable ZIP, we will use compression *and* GZIP.

Reply:
Yes, I agree, it's not very clear. I would prefer to have a single property, like: `oak.indexer.flatfilestore_compression = [none | gzip | lz4]`. But that is a bigger change, as we would have to update helm charts and bootstrap rules. Probably it's worth to do it to reduce technical debt sometime in the future, but we should not piggy back it in an unrelated PR.

Reply:
> But that is a bigger change, as we would have to update helm charts and bootstrap rules.

Hum, interesting, it's the first that see a _Helm chart_ being mentioned in relation with Oak.  
I'm curious: is there a such a chart which is publicly available?

Reply:
Not that I'm aware. My comment should not have mentioned helm charts, which is something we use internally, it should instead have been about breaking public APIs. I rephrased the comment. Thanks for the remark.